### PR TITLE
Fix crkbd slave matrix print to require debug_matrix

### PR DESCRIPTION
**Pull request by manna-harbour**
## Description
The crkbd prints slave matrix debug info with CONSOLE_ENABLE.  This change only prints with debug_matrix.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/rev1/split_scomm.c [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include "serial.h"
 #ifdef CONSOLE_ENABLE
   #include <print.h>
+  #include <debug.h>
```
Comment (fauxpark) [inline keyboards/crkbd/rev1/split_scomm.c:12]:
```suggestion
  #include "print.h"
  #include "debug.h"
```

Comment (drashna) [reply keyboards/crkbd/rev1/split_scomm.c:12]: Also, debug.h includes print.h, so shouldn't need both here, just one or the other.

File: keyboards/crkbd/rev1/split_scomm.c [hunk 2]
```diff
@@ -63,9 +64,11 @@ int serial_update_buffers(int master_update)
         if( smatstatus == TRANSACTION_END ) {
             s_change_old = s_change_new;
 #ifdef CONSOLE_ENABLE
-            uprintf("slave matrix = %b %b %b %b\n",
+            if (debug_matrix) {
+                uprintf("slave matrix = %b %b %b %b\n",
                     serial_slave_buffer[0], serial_slave_buffer[1],
                     serial_slave_buffer[2], serial_slave_buffer[3]);
+            }
```
Comment (drashna) [inline keyboards/crkbd/rev1/split_scomm.c:71]: dprintf should handle this automatically:

https://github.com/qmk/qmk_firmware/blob/d21730774752b8bc439564b1e3b687ad2aa0922c/tmk_core/common/debug.h#L67-L71


```suggestion
            dprintf("slave matrix = %b %b %b %b\n",
                serial_slave_buffer[0], serial_slave_buffer[1],	                    serial_slave_buffer[0], serial_slave_buffer[1],
                serial_slave_buffer[2], serial_slave_buffer[3]);	                    serial_slave_buffer[2], serial_slave_buffer[3]);
```

**Comment by drashna** [comment]
(misread: using debug_matrix not debug_print).

File: keyboards/crkbd/rev1/split_scomm.c (hunk 1 above)
Comment (manna-harbour) [reply keyboards/crkbd/rev1/split_scomm.c]: Thanks for the changes!

**Comment by zvecr** [comment]
Thanks!

**Comment by manna-harbour** [comment]
Thanks everyone!

**Pull request merged**